img_stats:
  rgb:
    mean: [0.485, 0.456, 0.406]
    std: [0.229, 0.224, 0.225]

  gray:
    mean: 0.3670
    std: 0.2932

  max_disparity: 100.0